airbyte-scheduler/app/Dockerfile

```diff
@@ -1,32 +1,10 @@
 FROM openjdk:14.0.2-slim AS scheduler
 
-
-# Install Docker to launch worker images. Eventually should be replaced with Docker-java.
-# See https://gitter.im/docker-java/docker-java?at=5f3eb87ba8c1780176603f4e for more information on why we are not currently using Docker-java
-RUN apt-get update && apt-get install -y \
-                          apt-transport-https \
-                          ca-certificates \
-                          curl \
-                          gnupg-agent \
-                          software-properties-common
-RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
-RUN add-apt-repository \
-       "deb [arch=amd64] https://download.docker.com/linux/debian \
-       $(lsb_release -cs) \
-       stable"
-RUN apt-get update && apt-get install -y docker-ce-cli jq
-
 ENV APPLICATION airbyte-scheduler
 
 WORKDIR /app
 
-# Install kubectl
-RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.17.14/bin/linux/amd64/kubectl
-RUN chmod +x ./kubectl
-RUN mv ./kubectl /usr/local/bin
-
-# Move and run scheduler
-COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
+COPY build/distributions/${APPLICATION}-0*.tar ${APPLICATION}.tar
```

Comment:
Is this expected? I don't see any change that builds the tar with a `-0` suffix.

Reply:
This is just the way we've done it in the past for the server, which is what I copied for this.

```
→ rg "APPLICATION}-0"
airbyte-server/Dockerfile
9:COPY build/distributions/${APPLICATION}-0*.tar ${APPLICATION}.tar

airbyte-scheduler/app/Dockerfile
7:COPY build/distributions/${APPLICATION}-0*.tar ${APPLICATION}.tar
```

The reason is that we are producing shadow jars for Maven / cloud consumption:
```
→ ll airbyte-server/build/distributions
total 1332472
-rw-r--r--  1 jrhizor  staff   175M Sep  7 14:34 airbyte-server-0.29.15-alpha.tar
-rw-r--r--  1 jrhizor  staff   158M Sep  7 14:34 airbyte-server-0.29.15-alpha.zip
-rw-r--r--  1 jrhizor  staff   166M Sep  7 14:35 airbyte-server-shadow-0.29.15-alpha.tar
-rw-r--r--  1 jrhizor  staff   149M Sep  7 14:35 airbyte-server-shadow-0.29.15-alpha.zip
```

Reply:
this will break once publish version 1.0 of our product. 😅

Reply:
Oh, got it. I did not realize that it was trying to match a `0.x` version.